trueline.sh

```diff
@@ -445,6 +445,24 @@ _trueline_cmd_duration_segment() {
     fi
 }
 
+_trueline_distro_icon_segment() {
+    case "$(cat /etc/os-release | grep ^ID= )" in
+        *ubuntu*) distro_icon=" " ;;
+        *fedora*) distro_icon=" " ;;
+        *debian*) distro_icon=" " ;;
+        *arch*) distro_icon=" " ;;
+        *) distro_icon="${TRUELINE_SYMBOLS[empty_distribution]} " ;;
```

Comment:
I removed the case where the output would be **Linux** because now we don't need it any more. Because in this case we will take a linux distribution in the **ID** row of the **os-release** file, but if we will not take something from our check here, we will simply output the default icon from the `TRUELINE_SYMBOLS` array.

Reply:
Can you make the default icon a the linux penguin instead of a coffee then?